### PR TITLE
Copy some rule changes over from OMD

### PR DESCRIPTION
**Pull request by joelmichael**
This reduces unnecessary cops, allowing us to focus on the things
which matter more.

### CODE REVIEW

File: default.yml [hunk 1]
```diff
@@ -105,6 +123,21 @@ RSpec/LetSetup:
...
+  Enabled: false
+
+RSpec/EmptyExampleGroup:
+  Enabled: false
+
+RSpec/MessageSpies:
+  Enabled: false
+
+RSpec/AnyInstance:
+  Enabled: false
```
Comment (nshoes) [inline default.yml]: This one always annoyed me 😆

File: default.yml [hunk 2]
```diff
@@ -47,6 +53,15 @@ Metrics/LineLength:
 Metrics/ModuleLength:
   Max: 200
 
+Metrics/ClassLength:
+  Max: 200
+
+Metrics/MethodLength:
+  Max: 20
```
Comment (nshoes) [inline default.yml]: How about 15 lines? 20 feels too long for me, what does everyone else think?

Comment (dinjas) [reply default.yml]: I agree that 20 seems long.

File: default.yml [hunk 3]
```diff
@@ -105,6 +123,21 @@ RSpec/LetSetup:
...
   Enabled: false
 
+RSpec/ImplicitBlockExpectation:
+  Enabled: false
+
+RSpec/MessageChain:
+  Enabled: false
+
+RSpec/EmptyExampleGroup:
+  Enabled: false
```
Comment (dinjas) [inline default.yml]: is there a reason this should be allowed? seems odd to have empty example groups unless they were just placeholders or something - easy enough to add a simple expectation

Comment (joelmichael) [reply default.yml]: We do have some empty spec files that were generated and have no specs. I'm willing to let this one go though... I do agree it makes sense that most spec files could probably have at least *one* spec written for them.

File: default.yml (hunk 2 above)
Comment (joelmichael) [reply default.yml]: I originally tried 15 but there were still some controller actions that it didn't seem to me were reasonable to break up into smaller methods. I suppose I'm willing to compromise on this issue a bit though.

**Comment by joelmichael** [comment]
@benjaminwood I will need you to re-review this after the version bump.

File: default.yml (hunk 2 above)
Comment (joelmichael) [reply default.yml]: I still think 20 is better, but have made it 15 for now.

**Pull request merged**